Request to merge Feature styling

Kingstalux opened the pull request:
- I added styling to the Bookstore App. 
- I used Zeplin as a collaboration tool while styling the book store app.
- A preview of the app has been added to the README file.
- A live link to the page has been added to the README file.

Kingstalux committed:
Add complete Nav bar

Kingstalux committed:
Add book card

Kingstalux committed:
Add nav bar styles

Kingstalux committed:
Add form styles

Kingstalux committed:
Add card styles

Kingstalux committed:
Edit styles

Kingstalux committed:
Add react icons

Kingstalux committed:
Edit readme and fix linters errors

Kingstalux committed:
Add live link and screen shot

Ghiftee commented on src/index.css:
```diff
@@ -1,3 +1,277 @@
+
+p.book-author {
+  background-color: white;
+  color: #0290ff;
```
- [x] This should have color #4386bf. Please refer to the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) for more information.

Ghiftee commented on src/index.css:
```diff
@@ -1,3 +1,277 @@
+}
+
+.btn {
+  color: #0290ff;
```
- [x] This should have color #4386bf. Please refer to the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) for more information.

Ghiftee commented on src/index.css:
```diff
@@ -1,3 +1,277 @@
+}
+
+.circle svg {
+  color: #0290ff;
```
- [x] This should have color #4386bf. Please refer to the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) for more information.

Ghiftee commented on src/index.css:
```diff
@@ -1,3 +1,277 @@
+
+p.current {
+  font-size: 1.125rem;
+  color: #888;
```
- [x] This should have color #e8e8e8 as per the Zeplin design

Kingstalux replied:
okay, I've implemented all color changes.

Ghiftee commented on src/index.css:
```diff
@@ -1,3 +1,277 @@
+
+p.completed {
+  font-size: 1.125rem;
+  color: #888;
```
- [x] This should have color #e8e8e8 as per the Zeplin design.

Ghiftee reviewed (changes requested):
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽 @Kingstalux 

Good job working on the project so far! 
✔️ Your design is neat
✔️ There are no broken images or links
✔️ Your linters pass
However, there are some issues that you need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] The add book button should be on the same line as the book title and category input as shown in the design. Please make the changes
- Your design
<img width="1204" alt="Screenshot 2021-11-18 at 12 16 59" src="https://user-images.githubusercontent.com/64774936/142407065-5bdc3998-ff99-4753-b4ed-74bbfad344d5.png">

- Zeplin design
<img width="1123" alt="Screenshot 2021-11-18 at 12 17 22" src="https://user-images.githubusercontent.com/64774936/142407202-0293535f-63e6-4c3b-8ba8-3535db1fcce8.png">

- [x] Your design currently misses the horizontal line between the completed percentage and the current chapter as shown in the image below. Also, the zeplin design has a half-circle but your design has a full one, kindly fix them

- Your design
<img width="663" alt="Screenshot 2021-11-18 at 12 20 44" src="https://user-images.githubusercontent.com/64774936/142407515-31237a30-485c-4588-8d11-24114c54b43e.png">

- Zeplin design
<img width="676" alt="Screenshot 2021-11-18 at 12 20 40" src="https://user-images.githubusercontent.com/64774936/142407620-c935dc03-6748-4832-887c-4e92305291ce.png">

- [x] Your design uses the Times font-family however, the Zeplin design uses Robotoslab and Montserrat. You can have a look at the [Zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) for more information on where to use which font family.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

Kingstalux commented:
- Hello @Ghiftee, thanks for the feedback, it was very helpful. So I've applied the following changes.
- I've corrected the colors and font families for the elements according to the designs
- I've added the vertical line in between the percentage and the chapter.
- I have aligned the add button with the inputs but depending on your screen resolution it might not look aligned since my resolution might be either bigger or smaller than yours. So maybe check on that.
- Finally for the circle, I took it from React font awesome icons, and only a full circle is found there. I tried exporting the half-circle from Zeplin but to no avail.
- Thats all for now, thanks.

Kingstalux committed:
Add requested changes

BrianSammit reviewed (approved):
# STATUS: APROVED :green_circle: :clap: 

Hello @Kingstalux, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

![well-1](https://user-images.githubusercontent.com/64173189/128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c.gif)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
